lib/usb_connection.js

```diff
@@ -14,7 +14,8 @@ var debug = require('debug')('discovery:usb');
 var DFU = require('./dfu');
 var logs = require('./logs');
 
-
+const outVendorcontrolTransferType = 0x40;
+const requestResetUSBD = 0x02;
```

Comment:
Yeah, I'm good with that. I'll update this PR before merging. Need to do some more testing first.